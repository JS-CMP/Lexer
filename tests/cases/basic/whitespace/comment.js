//       This is a comment with spaces

/*
   Block comment
   with spaces and newlines
*/
