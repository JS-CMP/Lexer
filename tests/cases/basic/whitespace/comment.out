#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
    return 0;
}
